### PR TITLE
Hide Managed Images link under RHEL Edge

### PR DESCRIPTION
**Pull request by boaz0**
Can you merge this to prod-stable as well?
CLOSE ticket [RHELPLAN-88724](https://issues.redhat.com/browse/RHELPLAN-88724)

//cc @omaciel @matoval @resoluteCoder @adeliaferreira

//cc2 @karelhala @Hyperkid123

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -406,6 +406,8 @@ edge:
         title: Fleet Management
         group: fleet-management
       - id: manage-images
+        permissions:
+          method: isHidden
```
Comment (Hyperkid123) [inline main.yml]: The `isHidden` is not permission but just a normal flag under nav items.

```yml
- id: manage-images
  isHidden: true
  title: Manage images
```
(if the validator fails add the attribute to the list in the validate-yaml.js file)

**Comment by Hyperkid123** [comment]
@boaz0 can you allow the attribute in the validator here: https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/validate-yaml.js#L19 ?

Just add `isHidden: Joi.boolean()`. It will fix the CI issue.

**Comment by boaz0** [comment]
Thanks @Hyperkid123 :bow: :1st_place_medal:

**Comment by boaz0** [comment]
got it :green_circle: @Hyperkid123 mind taking a second look :grinning: 
Thanks a lot for your help.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by boaz0** [comment]
Thanks!!!

**Pull request merged**